setup.sh

```diff
@@ -48,9 +48,9 @@ function install_go_bins() {
       "${LIGHT_GREEN}export GOPATH=\$GOPATH:~/code/golang${CLEAR}"
   fi
 
-  if [ -z $(which glide 2>/dev/null) ]; then
-    echo -e "${CLEAR}${LIGHT_GREEN}Installing glide${CLEAR}"
-    curl https://glide.sh/get | sh
+  if [ -z $(which dep 2>/dev/null) ]; then
+    echo -e "${CLEAR}${LIGHT_GREEN}Installing dep${CLEAR}"
+    curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
Are we using `master` or do we want to have it locked to last known working version?

Reply:
So, I cannot use the tag because the script wasn't there yet... I'll use the latest commit then...